cmd/copy.go

```diff
@@ -270,15 +270,21 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 			scanner := bufio.NewScanner(f)
 			for scanner.Scan() {
 				v := scanner.Text()
-				listChan <- v
+				// empty strings should be ignored, otherwise the source root itself is selected
```

Comment:
This just reminded me. Do we error out properly if the old format is handed in? I don't think so.

Reply:
The flag is internal so very few user should have visibility on it. Let's mention it in our change log.